test/shm_size.bats

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+  setup_test
+}
+
+function teardown() {
+  cleanup_test
+}
+
+@test "check /dev/shm is changed" {
+  # Start it with the environment variable set, don't use replace_config
+  CONTAINER_ENABLE_CUSTOM_SHM_SIZE=true start_crio
+
+  # Run base container to ensure it creates at all
+  run crictl runp <(cat "$TESTDATA"/sandbox_config.json | jq '.annotations."io.kubernetes.cri-o.ShmSize" = "16Mi"')
+  echo "$output"
+  [ "$status" -eq 0 ]
```

Comment:
You can replace these three lines with just
```bash
pod_id=$(crictl runp <(cat "$TESTDATA"/sandbox_config.json | jq '.annotations."io.kubernetes.cri-o.ShmSize" = "16Mi"'))
```

which does almost the same, including the exit code checking. The only different is it does not print stdout, only stderr.

Reply:
Sorry, I mean four lines, not three. Obviously it does pod_id assignment, too.

Reply:
Come to think of it, don't need all the echoes do we?